src/components/Workspace.jsx

```diff
@@ -206,20 +230,32 @@ class Workspace extends React.Component {
   }
 
   _renderOutput() {
-    const {hiddenUIComponents} = this.props.currentProject;
+    const {
+      currentProject,
+      currentProject: {hiddenUIComponents},
+      errors,
+      isMaskingOutput,
+      rowsFlex,
+      runtimeErrors,
+    } = this.props;
     return (
       <Output
-        errors={this.props.errors}
+        errors={errors}
         isHidden={includes(hiddenUIComponents, 'output')}
-        project={this.props.currentProject}
-        runtimeErrors={this.props.runtimeErrors}
+        project={currentProject}
+        runtimeErrors={runtimeErrors}
+        style={{
+          flex: rowsFlex[1],
+          pointerEvents: isMaskingOutput ? 'none' : 'all',
```

Comment:
It seems like this decision should be made by the `<Output>` component? Arguably.